src/borg/__init__.py

```diff
@@ -4,3 +4,13 @@
 
 
 __version_tuple__ = tuple(LooseVersion(__version__).version[:3])
+
+# assert that all semver components are integers
+# this is mainly to show errors when people repackage poorly
+# and setuptools_scm determines a 0.1.dev... version
+assert all(isinstance(v, int) for v in __version_tuple__), \
+    """\
+broken borgbackup version metadata: %r
+
+please ensure the real version metadata is availiable
```

Comment:
typo "available"

also: that "please ensure ..." is not very helpful if the reader does not know how to ensure that.